src/services/exit_order_iterator_state.py

```diff
@@ -20,7 +20,8 @@ class NodeOperatorPredictableState:
     delayed_validators_count: int
 
 
-class ExitOrderStateService(LidoValidatorStateService):
+class ExitOrderIteratorStateService(LidoValidatorStateService):
+    """Service to prepare state for exit order iterator"""
```

Comment:
Service prepares lido operator statistic, which used to form validators queue in right order